app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,86 @@
+?xml version="1.0" encoding="utf-8"?>
```

Comment:
First character '<' missing.
It should be: <?xml version="1.0" encoding="utf-8"?>